_tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md

```diff
@@ -479,15 +479,19 @@ Request parameters | Description
 `include_global_state` | Whether to restore the cluster state. Default is `false`.
 `include_aliases` | Whether to restore aliases alongside their associated indexes. Default is `true`.
 `partial` | Whether to allow the restoration of partial snapshots. Default is `false`.
-`rename_pattern` | If you want to rename indexes as you restore them, use this option to specify a regular expression that matches all indexes you want to restore. Use capture groups (`()`) to reuse portions of the index name.
-`rename_replacement` | If you want to rename indexes as you restore them, use this option to specify the replacement pattern. Use `$0` to include the entire matching index name, `$1` to include the content of the first capture group, and so on.
```

Comment:
Below: Either "that matches all the indexes that you want to restore and rename" or "that matches all the indexes that you want to rename."